openlibrary/core/lending.py

```diff
@@ -230,8 +230,12 @@ def get_groundtruth_availability(ocaid, s3_keys=None):
     including 1-hour borrows"""
     params = '?action=availability&identifier=' + ocaid
     url = S3_LOAN_URL % config_bookreader_host
-    r = requests.post(url + params, data=s3_keys)
-    data = r.json().get('lending_status')
+   try:
+        response = requests.post(url + params, data=s3_keys)
+        response.raise_for_status()
+    except requests.HTTPError:
+        pass  # TODO: Handle unexpected responses from the availability server.
```

Comment:
Line 237 was supposed to be implemented because line 238 will raise an exception if 237 was run.

Reply:
Can we patch with the previous:

```
    # TODO: This does not handle unexpected responses from the availability server.
    r = requests.post(url + params, data=s3_keys)
    data = r.json().get('lending_status')
```
for now and then open an issue?